test/unit-test/MQTTFileDownloader_config.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * AWS IoT Core MQTT File Streams Embedded C v1.1.0
+ * Copyright (C) 2023 Amazon.com, Inc. and its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: MIT
+ *
+ * Licensed under the MIT License. See the LICENSE accompanying this file
+ * for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/**
+ * @file MQTTFileDownloader_config.h
+ * @brief Testing values for various MQTT stream macros.
+ */
+
+#ifndef MQTT_FILE_DOWNLOADER_CONFIG_H
+#define MQTT_FILE_DOWNLOADER_CONFIG_H
+
+#include <stdint.h>
+
+#define mqttFileDownloader_CONFIG_BLOCK_SIZE    256U
```

Comment:
Perhaps the default should be specified so that the payload is 4k

Reply:
I'm fine with this. That would match the old OTA library better.